b2share/modules/deposit/loaders.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of EUDAT B2Share.
+# Copyright (C) 2016 CERN.
+#
+# B2Share is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# B2Share is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with B2Share; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""B2Share deposit input loaders."""
+
+from flask import abort, request
+from invenio_rest.errors import RESTValidationError, FieldError
+
+IMMUTABLE_PATHS= {
+    # fields added by the serializer
+    '/owners'
+    '/ePID_PID'
+    '/DOI',
+    # real fields
+    '/community',
+    '/$schema',
+    '/version',
+    '/_pid',
+    '/_files',
+    '/_deposit'
+    '/creators',
```

Comment:
Maybe I misunderstood the meaning of this `version` field then. I thought that it was some outside version number. It seems to be the case in V1.
If it is the version given by B2Share/Invenio then we should completely remove the field and provide the version number later at the same level as "_created" and "_updated", i.e. outside of the metadata.